docs/product/issues/issue-details/replay-issues/hydration-error.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Hydration Error
+sidebar_order: 50
+description: "Learn about hydration errors."
+---
+
+Hydration errors are a React-specific problem that happen when the initial UI does not match what was rendered on the server. They can result in extra work for the browser, and a slower pageload experience for users.
+
+## Prerequisites for Seeing Hydration Error Diff Tools
+
+Hydration Errors can come into your issue stream either as regular error events, or they could be detected inside a replay session and converted into an issue by the server. Read more about
+<PlatformLink to="/session-replay/issue-types/">configuring hydration errors</PlatformLink> to learn about the differences.
```

Comment:
```suggestion
[configuring hydration errors](https://docs.sentry.io/platforms/javascript/session-replay/issue-types/)o learn about the differences.
```

Changing this to a direct link so that it doesn't prompt you to choose JS when that is the only option